DataFormats/L1TParticleFlow/interface/gt_datatypes.h

```diff
@@ -185,12 +186,12 @@ namespace l1gt {
     z0_t seed_z0;
     ap_uint<1> charge;
     ap_uint<2> type;
-    iso_t isolation;
+    tau_rawid_t isolation;
```

Comment:
Just confirming this is a fix, which synchronises the Tau object definition to what's in the interface document?

Reply:
Hi @Duchstf ,@EmyrClement, Elias from the GT here, we were just gazing over these "tau_rawid_t". We were wondering if this parameter has any unit, and how it is intended to be "cut" on? 

We found this line: https://github.com/cms-l1t-offline/cmssw/blob/7c2e09fd7dfe4eef671b0e4cdcfa0fd87bc38aac/L1Trigger/Phase2L1ParticleFlow/plugins/L1NNTauProducer.cc#L290
and were wondering if these two are related and if the conversion is correct?

Reply:
I would like to add, that the in the menu we would want to select all taus with "passLooseNN()". Can you clarify this point too? Thank you @EmyrClement and @Duchstf